src/routes/guides/+page.svelte

```diff
@@ -21,16 +23,10 @@
     {
       title: "Defining Data Products",
       href: "https://www.youtube.com/watch?v=yPzN04ICsbw",
-      cardImageAlt: "QR code",
-      target: "_blank",
-      cardImage: VideoGuideIcon,
     },
     {
       title: "Productizing Data",
       href: "https://www.youtube.com/watch?v=f-f6P_-8zoQ",
-      cardImageAlt: "QR code",
-      target: "_blank",
-      cardImage: VideoGuideIcon,
     },
```

Comment:
If this list is called `secondaryCards` and when rendered there's a hard-coded `VideoGuideIcon` and `Watch`, this needs to be called `videoGuides` .. and why is it in this file and not the urls file?